website/package.json

```diff
@@ -17,12 +17,13 @@
     "@mdx-js/mdx": "1.1.0",
     "@mdx-js/react": "1.0.27",
     "common-tags": "1.8.0",
+    "change-case": "3.1.0",
```

Comment:
What's this for?

Reply:
Otherwise the `website` built failed cause MDX claimed it to be missing.

Reply:
Hmm ok, I'll take a look at upgrading gatsby-mdx as a follow up. Thanks for looking into it!